src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -793,6 +810,7 @@ internal StyleViewModel(OptionSet optionSet, IServiceProvider serviceProvider) :
             CodeStyleItems.Add(new BooleanCodeStyleOptionViewModel(CSharpCodeStyleOptions.PreferInferredTupleNames, ServicesVSResources.Prefer_inferred_tuple_names, s_preferInferredTupleName, s_preferInferredTupleName, this, optionSet, expressionPreferencesGroupTitle));
             CodeStyleItems.Add(new BooleanCodeStyleOptionViewModel(CSharpCodeStyleOptions.PreferInferredAnonymousTypeMemberNames, ServicesVSResources.Prefer_inferred_anonymous_type_member_names, s_preferInferredAnonymousTypeMemberName, s_preferInferredAnonymousTypeMemberName, this, optionSet, expressionPreferencesGroupTitle));
             CodeStyleItems.Add(new BooleanCodeStyleOptionViewModel(CSharpCodeStyleOptions.PreferLocalOverAnonymousFunction, ServicesVSResources.Prefer_local_function_over_anonymous_function, s_preferLocalFunctionOverAnonymousFunction, s_preferLocalFunctionOverAnonymousFunction, this, optionSet, expressionPreferencesGroupTitle));
+            CodeStyleItems.Add(new BooleanCodeStyleOptionViewModel(CodeStyleOptions.PreferNamedArguments, ServicesVSResources.Prefer_named_literal_arguments_CSharp_7_2_or_greater, s_preferNamedArguments, s_preferNamedArguments, this, optionSet, expressionPreferencesGroupTitle));
```

Comment:
@CyrusNajmabadi 
I discussed how to structure the options with DavidP.
We're thinking it may be better to stick with boolean options (rather than enum options), but have more than one (one for literals, one for lambdas, one for everything) when we want, under one section.
Then it would look like the section for "var" preferences:
![image](https://user-images.githubusercontent.com/12466233/30287461-0481c7e4-96da-11e7-8132-706435dfd50c.png)

In the meantime, it would be a single boolean option for literals.

This would also solve a technical problem. When I tried to fix the code above (since `BooleanCodeStyleOptionViewModel` because I pass an option that's wrong), the constructor for `EnumCodeStyleOptionViewModel` does not seem to handle per-language options, but only plain options.

Let me know if that approach seems ok.